core/commands/storage/session.go

```diff
@@ -41,42 +49,65 @@ func NewSessionID() (string, error) {
 	return seid.String(), nil
 }
 
-func (s *Session) NewSession(ssID string, fileHash string) error {
+func (s *Session) NewSession(ssID string) {
 	s.Lock()
 	defer s.Unlock()
 
-	ch := make(map[string]*StorageChallenge)
-	session := &Session{
-		Time:      time.Now(),
-		Status:    "init",
-		FileHash:  fileHash,
-		Challenge: ch,
-	}
-	SessionMap[ssID] = session
-	return nil
+	s.Time = time.Now()
+	s.Status = "init"
```

Comment:
note: make sure to update this time when you update status, otherwise the challenge timeout does not work since time is always from "init", not from "reqc"

Reply:
use chunk time instead of session time to check out challenge timeout, first will record time when setting a challenge

Reply:
yes, sure